src/categories/categories.controller.ts

```diff
@@ -17,11 +20,17 @@ export class CategoriesController {
     return this.categoriesService.findAll();
   }
 
+  @Get('getAllApis:id')
+  @ApiOperation({summary:"get all apis in a particular category"})
+  findAllApis(@Param('categoryId') categoryId: string) {
```

Comment:
u used :id as the route parameter and now using categoryId again. why?

Reply:
Mistake. It has been fixed